next.config.js

```diff
@@ -1,3 +1,8 @@
 const withCss = require("@zeit/next-css");
+require('dotenv').config();
```

Comment:
`dotenv` nicht lieber in Zeile 1 requiren?
Macht vermutlich keinen Unterschied, lese nur immer, dass es als Erstes importiert werden soll.